src/commands/functions-delete.ts

```diff
@@ -68,17 +68,15 @@ export default new Command("functions:delete [filters...]")
           return "\t" + helper.getFunctionLabel(func);
         })
         .join("\n");
-      confirmDeletion = await prompt(options, [
-        {
-          type: "confirm",
-          name: "confirm",
-          default: false,
-          message:
-            "You are about to delete the following Cloud Functions:\n" +
-            deleteList +
-            "\n  Are you sure?",
-        },
-      ]);
+      confirmDeletion = await promptOnce({
```

Comment:
The bug here was that prompt returns a map of {question.name: response}, so confirmDeletion was an object that looked like {"confirm": false}. This evaluates truthy, so the command would never abort.